### PR TITLE
Adding STAC and IIT mandi logos

### PR DESCRIPTION
**Pull request by abhijeetmanhas**
I have created logo section in index.html file. Check if it looks nice.

### CODE REVIEW

**Comment by shreyasbapat** [comment]
Can you post the screenshot of laptop as well as mobile view? This is directly deployed.

File: index.html [hunk 1]
```diff
@@ -75,6 +75,35 @@ <h2 style="color:white">29-31 March 2019</h2>
...
+    <div class="container">
+      <div class="col-md-8 col-md-offset-2 section-title">
+        <h2>Where?</h2>
+      </div>
+      <div id="row">
+        <div class="col-md-6 col-sm-6 team">
+          <div class="thumbnail"></div>> <img src="img/iit-logo.jpg" alt="...">
+            <div class="caption">
+              <h3><br>Venue</h3>
+              <p>This fest is organised at Indian institute of Technology, Mandi </p>
```
Comment (shreyasbapat) [inline index.html]:
```suggestion
              <p>This Astro-Meet is organised at Indian institute of Technology, Mandi </p>
```

Comment (shreyasbapat) [reply index.html]: By the way, why have we added extra lines? Please post a screenshot

File: index.html [hunk 2]
```diff
@@ -75,6 +75,35 @@ <h2 style="color:white">29-31 March 2019</h2>
...
+            </div>
+          </div>
+        </div>
+        <div class="col-md-6 col-sm-6 team"> <br><br>
+          <div class="thumbnail"> <img src="img/stac-logo.jpg" alt="..." class="team-img">
+            <div class="caption">
+              <h3><br> <br>Organising Club</h3>
+              <p> We, Space Technology and Astronomy Cell (STAC), <br> the Astronomy Club of IIT Mandi</p>
```
Comment (shreyasbapat) [inline index.html]: No need to add `We`

File: index.html (hunk 1 above)
Comment (abhijeetmanhas) [reply index.html]: I will post screenshot. Actually without it, both logos were not at same level

**Comment by abhijeetmanhas** [comment]
I have sent concerned screenshots in  email

File: index.html (hunk 2 above)
Comment (abhijeetmanhas) [reply index.html]: done

**Comment by shreyasbapat** [comment]
I suspected this! 
Sorry for not conveying things propeprly.

@abhijeetmanhas , the logos have to be added on the banner. 

@akshat157  Perhaps you could lend a helping hand?

**Comment by abhijeetmanhas** [comment]
screenshot 30
screenshot 29
I can't do better than this. I think i need to learn bootstrap before that. Should I commit this?

**Comment by shreyasbapat** [comment]
It's your branch @abhijeetmanhas ! You don't need my permission to commit anything :)

**Pull request closed without merging**